lxd/instance/drivers/driver_qemu.go

```diff
@@ -6492,7 +6492,17 @@ func (d *qemu) checkFeatures(qemu string) ([]string, error) {
 
 func (d *qemu) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error) {
 	if d.agentMetricsEnabled() {
-		return d.getAgentMetrics()
+		metrics, err := d.getAgentMetrics()
```

Comment:
@monstermunchkin @stgraber any reason we didn't do this previously?

Reply:
I cannot think of any reason we didn't use QEMU metrics as a fallback previously.